### PR TITLE
added Earth Eternal - Valkal's Shadow

### PR DESCRIPTION
**Pull request by rockfireredmoon**
Earth Eternal - Valkal's Shadow is a fan-made and run server for the abandoned MMORPG 'Earth Eternal'. The game is totally free to download and play, you just need to sign up for an account at  http://www.theanubianwar.com/user/register then use Request Key from your account page to create and link your game account. The game itself has worked flawlessly under wine for some time, as long as DirectX9 and Visual C++ runtime 2008 is installed.

### CODE REVIEW

File: Applications/Games/Earth Eternal - Valkal's Shadow/Online/script.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+	"scriptName": "Online",
+	"compatibleOperatingSystems": ["LINUX"],
+	"testingOperatingSystems": [],
+	"free": false,
```
Comment (madoar) [inline Applications/Games/Earth Eternal - Valkal's Shadow/Online/script.json]: I know I'm a little late but shouldn't this be `true`?

Comment (rockfireredmoon) [reply Applications/Games/Earth Eternal - Valkal's Shadow/Online/script.json]: Oh, yes it should. It is most definitely free. Do I need to do another pull request to correct this? 

plata - yes this is my first for POL 5. I made a script for the same game when it was active for classic PlayOnLinux, but this was definitely a lot easier. I've got to say, I'm loving the new approach taken and it's nice to see JavaFX being used too.

Comment (madoar) [reply Applications/Games/Earth Eternal - Valkal's Shadow/Online/script.json]: It's best to fix this by opening a new PR.
It's nice to hear other people like the new PlayOnLinux application :)

Comment (plata) [reply Applications/Games/Earth Eternal - Valkal's Shadow/Online/script.json]: Oh. Must have missed that. Anyways, it's not a big deal and can be fixed in a small pull request like @madoar suggested.

Glad that you like the new approach to writing scripts.

**Pull request merged**